server/plugin.go

```diff
@@ -878,19 +880,29 @@ func (p *Plugin) GetRemoteID() string {
 }
 
 func (p *Plugin) updateMetrics() {
-	start := time.Now()
-	p.API.LogDebug("Updating metrics")
-	stats, err := p.store.GetStats(p.remoteID, PreferenceCategoryPlugin)
+	now := time.Now()
+	p.API.LogInfo("Updating metrics")
+
+	stats, err := p.store.GetStats(storemodels.GetStatsOptions{
+		RemoteID:           p.remoteID,
+		PreferenceCategory: PreferenceCategoryPlugin,
+		ActiveUsersFrom:    now.Add(-metricsActiveUsersRange),
+		ActiveUsersTo:      now,
+	})
 	if err != nil {
 		p.API.LogWarn("failed to update computed metrics", "error", err)
 		return
 	}
+
 	p.GetMetrics().ObserveConnectedUsers(stats.ConnectedUsers)
 	p.GetMetrics().ObserveSyntheticUsers(stats.SyntheticUsers)
 	p.GetMetrics().ObserveLinkedChannels(stats.LinkedChannels)
 	p.GetMetrics().ObserveMattermostPrimary(stats.MattermostPrimary)
 	p.GetMetrics().ObserveMSTeamsPrimary(stats.MSTeamsPrimary)
-	p.API.LogDebug("Updating metrics done", "duration", time.Since(start))
+	p.GetMetrics().ObserveActiveUsersSending(stats.ActiveUsersSending)
+	p.GetMetrics().ObserveActiveUsersReceiving(stats.ActiveUsersReceiving)
+
+	p.API.LogInfo("Updating metrics done", "duration_ms", time.Since(now).Milliseconds())
```

Comment:
nit: I think we already get this from metrics for each SQL query. Is there value in logging separately?

Reply:
I don't run a grafana locally (maybe I should? :P) and I usually like to try those new queries against a bigger db, the log gives me a way to know how long it took. 0/5 to switch this to `LogDebug` if it makes more sense.

Reply:
> I don't run a grafana locally

Yes, definitely do this -- it's awesome! It's semi-automated with the monorepo if you're using Docker, but also trivial to spin up manually.

I'm leaning against adding one-off metrics as debug logs /if/ we have good ways of measuring them otherwise. (Log hygiene and all that.) But only 1/5 here.